app/upgrades/v19/utils.go

```diff
@@ -0,0 +1,78 @@
+package v19
+
+import (
+	"fmt"
+	"slices"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	"github.com/ethereum/go-ethereum/common"
+	evmostypes "github.com/evmos/evmos/v18/types"
+)
+
+func GetAccountBytecode(ak authkeeper.AccountKeeper, ctx sdk.Context, hexAddress string) common.Hash {
+	acc, err := sdk.AccAddressFromHexUnsafe(hexAddress)
+	if err != nil {
+		panic(err)
```

Comment:
Is the panic intentional? Can't we just log hexAddress that returned an error and remove later?